package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "open-city-profile-ui",
-  "version": "1.1.12",
+  "version": "1.1.11",
```

Comment:
Is it okay to go back in version numbers?

Reply:
We don't do package releases and version number is only used to for output.